File: Data Structures & Algorithms/longest-palindromic-substring/submission-0.cpp
class Solution {
public:
    string longestPalindrome(string s) {
        int res=-1;
        int resLen=0;
        for(int i=0;i<s.length();i++){
            int l=i,r=i;
            while(l>=0 && r<s.length() && s[l]==s[r]){
                if(r-l+1>resLen){
                    resLen=r-l+1;
                    res=l;
                }
                l--;
                r++;
            }
                l=i,r=i+1;
                while(l>=0 && r<s.length() && s[l]==s[r]){
                if(r-l+1>resLen){
                    resLen=r-l+1;
                    res=l;
                }
                l--;
                r++;
            }
        }
        return s.substr(res,resLen);
    }
};

File: Data Structures & Algorithms/plus-one/submission-0.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        carry=1
        res=[0]*(len(digits))
        for i in range(len(digits)-1,-1,-1):
            res[i]=(carry+digits[i])%10
            carry=(carry+digits[i])//10
        
        if carry==1:
            res.insert(0,1)

        return res
        